CaloReco/src/CaloHitMakerFast_module.cc

```diff
@@ -0,0 +1,158 @@
+#include "art/Framework/Core/EDProducer.h"
```

Comment:
What is the intent of this module?  Will it replace CaloHitMaker?  Or is it just used in the trigger sequence?

Reply:
this module is used to emulate what we do in `DAQ/src/CaloHitsFromFragments_module.cc`

Reply:
Giani should have a look at it and let me know if he wants changes (I had to guess a few things).

Reply:
I think that it's OK, **BUT** I'll probably refactor part of it in order to use the same function that is used in the DAQ/src/CaloHitsFromFramgents_module.cc I wrote in my "ghost"[PR](https://github.com/Mu2e/Offline/blob/67f614de2b8486465aad88ee4e169833cdc60444/DAQ/src/CaloHitsFromFragments_module.cc). 
Essentially I added setters function in the CaloHit class to avoid defining a custom temporary struct... which is more clean and also more efficient memory-wise.